geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/CreateDataSourceCommand.java

```diff
@@ -139,7 +139,10 @@ public ResultModel createJDNIBinding(
       return result;
     } else {
       if (service != null) {
-        return ResultModel.createInfo("No members found, data source saved to cluster config.");
+        ResultModel result =
+            ResultModel.createInfo("No members found, data source saved to cluster config.");
```

Comment:
The problem I have with saying "data source saved to cluster config" is it might not happen. I suggest only adding this to the message here if "service" is not null. Even in that case should we say here: "will attempt to save to cluster configuration" instead? (Note: change "config" to "configuration" in customer facing message).
We will ask gfsh to make the attempt and it will follow it up with its own message telling us if it was saved to cluster config. It is not until "updateConfigForGroup" is called and returns true that we know the cluster config was actually saved.

Reply:
Oops, I just saw we do check "service != null. So I'd be satisfied if you just change "config" to "configuration".

Reply:
I updated this to use the term configuration rather than the abbreviation.